src/EditorFeatures/Core.Wpf/SymbolSearch/SymbolSearchUpdateEngine.cs

```diff
@@ -64,7 +60,7 @@ internal partial class SymbolSearchUpdateEngine : ISymbolSearchUpdateEngine
             IPatchService patchService,
             IDatabaseFactoryService databaseFactoryService,
             Func<Exception, bool> reportAndSwallowException,
-            CancellationToken updateCancellationToken)
+            CancellationToken testCancellationToken)
```

Comment:
this is the constructor called from teh above one, and directly in tests.  tests are the only thing that actually pass a 'real' cancellation token here so that hte test can gracefully bail out once it has verified certain behavior.